README.md

```diff
@@ -3,6 +3,8 @@
 [![All Contributors](https://img.shields.io/badge/all_contributors-26-green.svg)](#contributors)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
+### This repository was clone from https://github.com/VictorCazanave/svg-maps. We would like to add a map, but the author is not responding. 
```

Comment:
For the past few years, I was too busy to maintain this project. Feel free to fork it and create your own package.